core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -396,6 +413,7 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 	}
 	cancel()
 
+	observeTimeUntilBroadcast(eb.chainID, etx.CreatedAt, time.Now())
```

Comment:
Update on this thread: Sam and I have synced offline on this, and since we can re-enter `handleInProgressEthTx` by way of `tryAgainBumpingGas`, and since we want to record the first _successful_ broadcast, we use `time.Now()` instead of `initialBroadcastAt`.